plate_mapper/plate_mapper.py

```diff
@@ -97,20 +101,50 @@ def plate_mapper(input_f, barseq_f, output_f, names_f=None):
             barseqs.append(line.split('\t'))
     barseq_f.close()
     print('  Done.')
+
+    # Read special sample definitions
+    specs = {}
+    if special_f:
+        print('Reading special sample definitions...')
+        next(special_f)  # skip header line
```

Comment:
We had this discussion before (see PR [#3](https://github.com/biocore/wetlab-assistant/pull/3) and issue [#8](https://github.com/biocore/wetlab-assistant/issues/8)). The idea here is to create a minimalist script that works right away on any computer, without installing any non-built-in Python packages, such as **click** and **pandas**. Note that the script is to be used by the wet lab crew, where any sophiscated setups such as typing "pip install..." has to be assisted by a dry lab person. Currently, the way the user uses this script is to double-click an icon (without even opening the terminal window or typing anything). I hope to keep this easiness.